zio-http/shared/src/main/scala/zio/http/FormField.scala

```diff
@@ -159,6 +159,8 @@ object FormField {
       ) {
         case (accum, header: FormAST.Header) if header.name.equalsIgnoreCase("Content-Disposition")       =>
           (Some(header), accum._2, accum._3, accum._4)
+        case (accum, FormAST.EoL)                                                                         =>
+          (accum._1, accum._2, accum._3, accum._4 :+ FormAST.Content(Chunk[Byte](13, 10)))
```

Comment:
```suggestion
          (accum._1, accum._2, accum._3, accum._4 :+ FormAST.Content(FormAST.EoL.bytes))
```

To prevent Chunk creation every time, I suggest re-use it from `FormAST.EoL`